packages/ember-htmlbars/lib/keywords/mut.js

```diff
@@ -63,11 +63,32 @@ let MutStream = ProxyStream.extend({
   // my-child.js
   export default Component.extend({
     click() {
-      this.attrs.childClickCount.update(this.attrs.childClickCount.value + 1);
+      this.get('childClickCount').update(this.get('childClickCount').value + 1);
     }
   });
   ```
 
+  Additionally, the `mut` helper can be combined with the `action` helper to more
```

Comment:
remove "more easily". "particuparlly" has a typo.
on a personal sidenote, I'm not sure it's useful to mention DDAU, especially in API docs